Makefile

```diff
@@ -125,6 +110,22 @@ prepare-byoh-docker-host-image:
 prepare-byoh-docker-host-image-dev:
 	docker build test/e2e -f docs/BYOHDockerFileDev -t ${BYOH_BASE_IMG_DEV}
 
+
+# Run tests
+test: generate fmt vet manifests test-coverage
+
+test-coverage: docker-build prepare-byoh-docker-host-image
```

Comment:
You don't need `docker-build` It is generating the controller manager image. For this test, probably just `prepare-byoh-docker-host-image` is enough